Gaijinizer patch/CommonEvents_275.txt

```diff
@@ -4,31 +4,35 @@
 
 大人しく、私と逝け……
 > CONTEXT: CommonEvents_275/13/Dialogue/enemy4
-You are already over...It's mediocre, it makes
-me ...#MTLed
+You are already finished......
+
+Be quiet, and die with me......#Strate
 > END STRING
 
 > BEGIN STRING
 どうした…？
 
 見逃して欲しいのか…？
 > CONTEXT: CommonEvents_275/20/Dialogue/enemy4
-What happened…? Do you want me to miss it?#MTLed
+What happened...?
+
+Do you want me to let you leave?#Strate
 > END STRING
 
 > BEGIN STRING
 そんな要求…
 
 通ると思っているのか…？
 > CONTEXT: CommonEvents_275/32/Dialogue/enemy4
-Such a request...Do you think that it passes
-...?#MTLed
+Such a request...
+
+Do you think that will work...?#Strate
 > END STRING
 
 > BEGIN STRING
 ………話すことなどない。
 > CONTEXT: CommonEvents_275/39/Dialogue/enemy4
-......I do not speak.#MTLed
+.........I will not say anything.#Strate
```

Comment:
Another funny one that doesn't translate well. I'm assuming this is after she rejects your persuasion, so probably closer to "There's nothing more to discuss" or "I won't hear any more on this matter"

Reply:
This one is actually the one where she accepts your persuasion during the indecision step. It's more like "I won't speak of this" as she lets you leave. Not sure how to convey that feeling accurately.

Reply:
I realized something, wouldn't it make more sense for her to be giving an order instead? Like "do not speak of this in the future"? I changed it to that effect for now but I'll leave this open just in case.

Reply:
I posted it on Hinative to double-check, and the answer I got back from a native speaker was "I don't want to talk with you", so perhaps some fitting variation on that?

Reply:
That's kinda difficult to fit into the context of her letting you go.  Maybe something like "No more talk, just leave"?

Reply:
I think that works.